packages/@aws-cdk/aws-events/lib/matcher.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Event Pattern matcher.
+ */
+export abstract class Matcher {
+
+    /**
+     * Matches event based attribute value.
+     * @param values possible values to match the event attribute against
+     */
+    public static match(...values: string[]): any[] {
```

Comment:
Alternatively instead of prefixing each method with `match` I was thinking about using just the matcher name.

This way `matchPrefix()` would become `prefix()`, `matchAnythingBut()` would be `anythingBut()` and so on.

Reply:
If you rename the class as well, I think `Match.prefix()` would look pretty neat.